loinc-integration/src/test/java/dev/ikm/tinkar/loinc/integration/LoincAxiomSemanticIT.java

```diff
@@ -28,7 +29,7 @@
 import java.util.concurrent.atomic.AtomicInteger;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-
+@Disabled
 @TestInstance(Lifecycle.PER_CLASS)
 public class LoincAxiomSemanticIT extends LoincAbstractIntegrationTest {
```

Comment:
Test classes are usually named `*Test.java` with `IT` as a prefix, if we are trying to distinguish from unit tests.